tests/freqai/test_freqai_interface.py

```diff
@@ -85,6 +86,21 @@ def test_extract_data_and_train_model_Standard(mocker, freqai_conf, model, pca,
     if 'test_3ac' in model or 'test_4ac' in model:
         freqai_conf["freqaimodel_path"] = str(Path(__file__).parents[1] / "freqai" / "test_models")
 
+    if 'PyTorchClassifierMultiTarget' in model:
+        model_save_ext = 'zip'
+        freqai_conf['freqai']['model_training_parameters'].update({
+            "n_hidden": 1024,
```

Comment:
Looks good, but one point is that we need these tests to be super short (freqai already takes a disproportionate time to test), so we should bring these parameters down to their bare minimum. So maybe 64 hidden_dim, batch_size 4? Iters 5?